website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Installing the dbt Cloud CLI (Alpha)
+id: cloud-cli-installation
+description: "Instructions for installing and configuring dbt Cloud CLI"
+---
+
+:::warning Alpha functionality 
+
+The following installation instructions are for the dbt Cloud CLI, currently in alpha. These instructions are not intended for general audiences at this time. 
+
+::: 
+
+## Installing dbt Cloud CLI
+
+### Install and update with Brew (recommended)
+
+Install the CLI: 
+
+```bash
+brew tap dbt-labs/dbt-cli
+brew install dbt-cloud-cli
+```
+
+Verify the installation:
+
+```bash
+which dbt
+dbt --help
+```
+
+Upgrade the CLI:
+
+```bash
+brew update
+brew upgrade dbt-cloud-cli
+```
+
+### Manually install (Windows only)
+
+1. Download the latest darwin release from [Github](https://github.com/dbt-labs/dbti/releases).
```

Comment:
"Download the latest release for your platform from [Github](https://github.com/dbt-labs/dbt-cli/releases)."

Note that the url changed